src/borg/helpers/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""
+This package contains all sorts of small helper / utility functionality,
+that did not fit better elsewhere.
+
+Code used to be in borg/helpers.py but was split into the modules in this
+package, which are imported into here for compatibility.
+"""
+
+from .checks import *  # NOQA
+from .convert import *  # NOQA
+from .datastruct import *  # NOQA
+from .errors import *  # NOQA
+from .fs import *  # NOQA
+from .manifest import *  # NOQA
+from .misc import *  # NOQA
+from .msgpack import *  # NOQA
+from .parseformat import *  # NOQA
+from .process import *  # NOQA
+from .progress import *  # NOQA
+from .time import *  # NOQA
+from .usergroup import *  # NOQA
+from .yes import *  # NOQA
```

Comment:
all names from the modules are imported into helpers.

with `__all__` and a `from borg.helpers import *` one would get borg.progress.*, which is different.